### ISSUE TITLE
Entry for the repeater field in the ACF cookbook outdated and misleading? 

### ISSUE THREAD

**Issue by rpkoller** [jarednova/timber#669]
I was following the lead from the ACF Cookbook to implement the output for a repeater field. Based on that snippet: 

```twig
{% for item_outer in post.get_field('outer') %}
     {{item_outer.title}}
     {% for item_inner in item_outer.inner_repeater %}
          {{item_inner.title}}
     {% endfor %}
{% endfor %}
```
and that snippet from the Cookbook: 

```twig
{% for gear in post.get_field('gear_items') %}
    <h3> {{ gear.brand_name }} </h3>  
    {% for gear_feature in gear.features %}
        <li> {{gear_feature}} </li>
    {% endfor %}
{% endfor %}
```

i've tried the following: 
```twig
{% for item_outer in post.get_field('workshop_teachers') %}
      {{ item_outer|print_r}}
      {% for item_inner in item_outer.workshop_teachers_id %}
           {{ item_inner|print_r}}
           <li itemprop="performer">{{ item_inner }}</li>
      {% endfor %}
{% endfor %}
```

The `item_outer` returned: 

```js
Array ( [workshop_teachers_id] => John Doe )
```

But the second had no output… Never. Then i've tried to leave out the second inner for loop and applied just the following: 

```twig
{% for item in post.get_field('workshop_teachers') %}
      <li itemprop="performer">{{ item.workshop_teachers_id }}</li>
{% endfor %}
```

With that the repeater output behaved as expected. So the wiki entry was somehow misleading. Shall i update it or do you want to? Best regards r.

**Comment by jarednova** [comment]
Hey @rpkoller as I understand from your example you have a single repeater field like so....

```php
'workshop_teachers' => Array(
    'workshop_teachers_id' => 'John Doe',
);
```

The example you cite from the cookbook is about _nested_ repeater fields, take a look at [this example](https://github.com/jarednova/timber/wiki/ACF-Cookbook#repeater-field) from the Cookbook:
```twig
{# single.twig #}
<h2>{{post.title}}</h2>
<div class="my-list">
    {% for item in post.get_field('my_repeater') %}
        <div class="item">
            <h4>{{item.name}}</h4>
            <h6>{{item.info}}</h6>
            <img src="{{TimberImage(item.picture).src}}" />
        </div>
    {% endfor %}
</div>
```

Does that better capture the structure of your data?

**Comment by rpkoller** [comment]
@jarednova thanks! I have to admit i am still too close to the PHP loop thinking i am used to from WordPress. There i had two loops for a single repeater field in ACF. Therefor i automatically thought the example in the Cookbook starring two for-loops had to be applied to my case as well. I overread the first example you've reposted in here. I've filed it under general example as an introduction to the field and haven't closely examined it and directly went on to the next code snippets which all dealt with the repeater in repeater examples you've mentioned. Figured both cases out now. The more i understand things the more i adore the way timber deals with loops and acf specific repeater fields in a clean and brief way. double thumbs up!
